jenkins/Jenkinsfile.linux

```diff
@@ -414,17 +414,39 @@ pipeline {
         }
       }
 
-      steps {
-        build wait: false,
-          job: "IDE/qa-${IS_PRO ? '' : 'opensource-'}automation",
-          parameters: [
-            string(name: 'RSTUDIO_VERSION_MAJOR', value: "${RSTUDIO_VERSION_MAJOR}"),
-            string(name: 'RSTUDIO_VERSION_MINOR', value: "${RSTUDIO_VERSION_MINOR}"),
-            string(name: 'RSTUDIO_VERSION_PATCH', value: "${RSTUDIO_VERSION_PATCH}"),
-            string(name: 'RSTUDIO_VERSION_SUFFIX', value: "${RSTUDIO_VERSION_SUFFIX}"),
-            string(name: 'SLACK_CHANNEL', value: "${params.SLACK_CHANNEL}"),
-            string(name: 'BRANCH_NAME', value: "${env.BRANCH_NAME}")
-          ]
+      stages {
+
+        stage('Trigger Automation Testing') {
+          steps {
+            build wait: false,
+            job: "IDE/qa-${IS_PRO ? '' : 'opensource-'}automation",
+            parameters: [
+              string(name: 'RSTUDIO_VERSION_MAJOR', value: "${RSTUDIO_VERSION_MAJOR}"),
+              string(name: 'RSTUDIO_VERSION_MINOR', value: "${RSTUDIO_VERSION_MINOR}"),
+              string(name: 'RSTUDIO_VERSION_PATCH', value: "${RSTUDIO_VERSION_PATCH}"),
+              string(name: 'RSTUDIO_VERSION_SUFFIX', value: "${RSTUDIO_VERSION_SUFFIX}"),
+              string(name: 'SLACK_CHANNEL', value: "${params.SLACK_CHANNEL}"),
+              string(name: 'BRANCH_NAME', value: "${env.BRANCH_NAME}")
+            ]
+          }
+        }
+
+        stage('Trigger Desktop Automation Testing') {
```

Comment:
Names are hard. Is this accurate? As opposed to the more general automation testing job above

Reply:
I think so -- the automation run here is specifically to run our existing test suite against desktop builds of RStudio.